.github/workflows/main.yml

```diff
@@ -31,6 +31,14 @@ jobs:
             tensorflow-version: 2.5.0
         notebook-details:
           - { dep: "huggingface nltk", extra: "'tensorflow>=2.5.0,<2.8.0,termcolor>=1.1.0'"}
+        modules:
+          - { module: "models", test_file: "tests/forte/processors/subword_tokenizer_test.py tests/forte/processors/pretrained_encoder_processors_test.py"}
+          - { module: "data_aug", test_file: "tests/forte/processors/subword_tokenizer_test.py tests/forte/processors/pretrained_encoder_processors_test.py"}
+          - { module:  "nlp",test_file: "tests/forte/models"}
+          - { module: "ir",test_file: "tests/forte/processors/ir/"}
+          - { module: "stave",test_file: "tests/forte/data/readers/stave_reader_test.py tests/forte/processors/stave_processor_test.py"}
+          - { module: "audio_ext",test_file: "tests/forte/data/readers/audio_reader_test.py"}
+          - { module: "extractor",test_file: "tests/forte/train_preprocessor_test.py forte/data/extractors"}
```

Comment:
I also don't see `remote` here, can you double check on all the test cases to make sure these are what you want?

Reply:
I have fixed all test files and there is not test file for remote processors.

Reply:
@mylibrar do we miss a test case for `remote`? Do we need to make an issue to track it?

Reply:
Test cases for `remote` are included in https://github.com/asyml/forte/blob/master/tests/forte/advanced_pipeline_test.py